SU2_CFD/src/output/CFlowCompFEMOutput.cpp

```diff
@@ -115,8 +115,9 @@ void CFlowCompFEMOutput::SetHistoryOutputFields(CConfig *config){
   AddHistoryOutput("MAX_ENERGY",     "max[RhoE]", ScreenOutputFormat::FIXED,   "MAX_RES", "Maximum residual of the energy.", HistoryFieldType::RESIDUAL);
   /// END_GROUP
 
-  /// DESCRIPTION: Linear solver iterations
-  AddHistoryOutput("LINSOL_ITER", "Linear_Solver_Iterations", ScreenOutputFormat::INTEGER, "LINSOL_ITER", "Number of iterations of the linear solver.");
+  /// DESCRIPTION: Linear solver iterations and residual
```

Comment:
I think these can be removed from the DG solver output, as it is entirely explicit in the current implementation

Reply:
Thought so too, but maybe they were being recycled for something else, they're gone now.